src/js/components/FileInput/FileInput.js

```diff
@@ -121,6 +122,7 @@ const FileInput = forwardRef(
       >
         <ContentsBox
           ref={controlRef}
+          tabIndex={!plain ? 0 : undefined}
```

Comment:
Would `plain` come into play when file input is in a form field? just wanting to understand when we wouldn't want it to have tabIndex 0

Reply:
yes

Reply:
nice!